_sass/kakapo.scss

```diff
@@ -0,0 +1,176 @@
+@import "kakapo/config";
+
+*{
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+html, body{
+
+}
+body{
+  font-family: "Open Sans", Helvetica, sans-serif;
+}
+a{
+  color: inherit;
+  text-decoration: none;
+  &:visited{
+    color: inherit;
+  }
+  &:hover{
+    color: inherit;
+    text-decoration: none;
+  }
+}
+ul{
+  list-style: none;
+  margin: 0;
+}
+
+.kakapo{
+  height: 50px;
+}
+
+#banner{
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(black, 0.5);
+  color: white;
+
+  h1{
+    font-size: 55px;
+  }
+  .motto-line{
+    width: 0;
+    height: 2px;
+    background-color: white;
+    margin: 0 auto;
+    animation: motto-line-animation 1.5s 0.2s cubic-bezier(0.31, 0.91, 1, 1) forwards;
+  }
+  .motto{
+    font-size: 25px;
+    margin-top: 10px;
+  }
+  .learn-more{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 50px;
+    word-spacing: 3px;
+    letter-spacing: 4px;
+    animation: learn-more-animation 1s 2s forwards;
+    opacity: 0;
+
+    &:hover{
+      .arrow{
+        transform: translateY(5px);
+      }
+    }
+    .arrow{
+      width: 25px;
+      height: 30px;
+      margin-top: 15px;
+      transition: transform .2s;  
+    }
+  }
+  &.js-banner:after{
+    background-image: url('/assets/images/js-bg.jpg');
```

Comment:
can we take this out, put it in the layout (documentation) and use a variable for the image name?